app/sessionStore.js

```diff
@@ -784,8 +784,9 @@ module.exports.loadAppState = () => {
 
       immutableData = module.exports.runPostMigrations(immutableData)
     }
-    immutableData = setVersionInformation(immutableData)
```

Comment:
this had to be moved; translations are only available AFTER the `locale` finishes initializing 😄